src/coresyftools/coresyftools/tool_tester.py

```diff
@@ -64,7 +65,7 @@ def __init__(self, tool_dir, scihub_credentials):
         self.scihub_credentials = scihub_credentials
         self.cwd = os.getcwd()
         examples_file = 'examples.sh'
-        self.manifest_file_name = os.path.join(tool_dir, 'manifest.json')
+        self.manifest_file_name = glob.glob(os.path.join(tool_dir, '*manifest.json'))[0]
```

Comment:
Since this is used in different classes I would put it in a function like `find_manifest_file` or `get_manifest_file` of `manifest.py` module.